01_prepare_host.sh

```diff
@@ -84,13 +99,17 @@ function init_minikube() {
     #If the vm exists, it has already been initialized
     if [[ "$(sudo virsh list --all)" != *"minikube"* ]]; then
       sudo su -l -c "minikube start" "$USER"
-      # The interface doesn't appear in the minikube VM with --live,
+      # The interfaces don't appear in the minikube VM with --live,
       # so just attach it and make it reboot. As long as the
-      # 02_configure_host.sh script does not run, the provisioning network does
-      # not exist. Attempting to start Minikube will fail until it is created.
+      # 02_configure_host.sh script does not run, the provisioning and
+      # baremetal networks do not exist. Attempting to start Minikube will fail
+      # until it is created.
       sudo virsh attach-interface --domain minikube \
           --model virtio --source provisioning \
           --type network --config
+      sudo virsh attach-interface --domain minikube \
+          --model virtio --source baremetal \
+          --type network --config
```

Comment:
Do you know why this was not needed before?  I don't have this, yet I'm using the baremetal network to ssh to my provisioned machines.

```
$ sudo virsh net-dhcp-leases baremetal
 Expiry Time          MAC address        Protocol  IP address                Hostname        Client ID or DUID
-------------------------------------------------------------------------------------------------------------------
 2019-09-30 15:57:38  00:8d:ab:f6:f2:5b  ipv4      192.168.111.21/24         worker-0        -

 $ fssh centos@192.168.111.21
Warning: Permanently added '192.168.111.21' (ECDSA) to the list of known hosts.
Last login: Mon Sep 30 19:16:41 2019 from gateway
[centos@worker-0 ~]$ 
```

Reply:
I'll investigate to see exactly why it was needed, but Ironic when running in the cluster couldn't reach vbmc without it

Reply:
I'll investigate to see exactly why it was needed, but Ironic when running in the cluster couldn't reach vbmc without it

Reply:
I definitely need the interface for Ironic in the cluster to talk to vbmc.  Otherwise Ironic says:

```
| last_error             | Failed to get power state for node c069fe7d-6d38-407e-9206-8745f19df274. Error: IPMI call failed: power status.                                                                                                                                                                       ```

When I do this from the Ironic container:

```
[root@minikube /]# ipmitool -I lanplus -H 192.168.111.1 -p 6230 -Uadmin -Ppassword power status
```

The request comes over the interface, I see it with tcpdump, but vbmc never replies...

```
09:23:02.541681 IP (tos 0x0, ttl 64, id 41227, offset 0, flags [DF], proto UDP (17), length 51)
    192.168.122.224.50637 > 192.168.111.1.6230: UDP, length 23
09:23:02.541701 IP (tos 0x0, ttl 64, id 41227, offset 0, flags [DF], proto UDP (17), length 51)
    192.168.122.224.50637 > 192.168.111.1.6230: UDP, length 23
```